server/cmd/mmctl/commands/post_e2e_test.go

```diff
@@ -192,3 +192,43 @@ func (s *MmctlE2ETestSuite) TestPostCreateCmd() {
 		s.Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestPostDeleteCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.Run("Delete a post for System Admin Client", func() {
+		printer.Clean()
+
+		postID := "some-post-id"
+
+		cmd := &cobra.Command{}
+
+		err := postDeleteCmdF(s.th.SystemAdminClient, cmd, []string{postID})
+		s.Require().Nil(err)
```

Comment:
I'm confused. "some-post-id" is not a valid postID. Why doesn't this call fail?

Reply:
Should i use createNewChannelAndPosts like TestPostListCmd and then create post and channel and then delete the post ?